xmlpoke/xmlpoke.d.ts

```diff
@@ -0,0 +1,45 @@
+// Type definitions for xmlpoke 0.1.12
+// Project: https://github.com/mikeobrien/node-xmlpoke
+// Definitions by: Garth Kidd <https://github.com/garthk/>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference path="../xpath/xpath.d.ts" />
+
+declare module XmlPoke { // ghost module
```

Comment:
Confusingly, `declare module XmlPoke` is a synonym for `declare namespace XmlPoke`, but `declare module 'xmlpoke'` declares an ES6 module.